README.md

```diff
@@ -88,13 +89,71 @@ Additionally, for the development we use:
 
 > See the [Development](#development) section for details on how to install `mockery`.
 
-# Installation
+# Quick-Start Installation
 
-## From binaries
+We provide an installation script to automatically install and update the latest version of Trento.
+
+## Trento Server Installation
+
+T.B.D.
+
+## Trento Agent Installation
+
+You can `curl | bash` if you want to live on the edge.
+
+```
+$ curl -sfL https://raw.githubusercontent.com/trento-project/trento/main/install.sh | sudo bash
+```
+
+Or you can fetch the script, and then execute it manually.
+
+```
+$ curl -O https://raw.githubusercontent.com/trento-project/trento/main/install.sh
+$ chmod 700 install.sh
+$ sudo ./install.sh
+```
+
+The script will ask you for two IP addresses.
+
+- `agent bind IP`: the private address to which the trento-agent should be bound for internal communications.
+  This is an IP address that should be reachable by the other nodes, including the trento server.
+  Note for the Pacemaker users: this IP address _should not be_ a floating IP.
+
+- `trento server IP`: the address where Trento server can be reached.
+
+You can pass these arguments as flags or env variables too:
+
+```
+$ curl -sfL https://raw.githubusercontent.com/trento-project/trento/main/install.sh | sudo bash -s - --agent-bind-ip=192.168.33.10 --server-ip=192.168.33.1
```

Comment:
When I tested this on SLES15 SP2 this happened:

```
cloudadmin@vmiscsisrv:~> curl -sfL https://raw.githubusercontent.com/fabriziosestito/trento/agent-installer/install.sh |sudo bash -s - --agent-bind-ip=10.75.1.4 --server-ip=10.0.0.4
Installing trento-agent...
warning: Rebuilding outdated index databases
warning: Generating 18 missing index(es), please wait...
* Installing trento
Done.
You can now start trento-agent with: systemctl start trento-agent

cloudadmin@vmiscsisrv:~> sudo bash
vmiscsisrv:/home/cloudadmin # systemctl start trento-agent
Failed to start trento-agent.service: Unit trento-agent.service is not loaded properly: Exec format error.
See system logs and 'systemctl status trento-agent.service' for details.

vmiscsisrv:/home/cloudadmin # systemctl status trento-agent
● trento-agent.service - Trento agent service
   Loaded: error (Reason: Exec format error)
   Active: inactive (dead)

Sep 10 16:32:27 vmiscsisrv systemd[1]: /usr/lib/systemd/system/trento-agent.service:7: Executable path is not absolute: trento agent start --consul-config-dir=/srv/consul/consul.d /srv/trento/examples/generic-azure.yaml /srv/trento/examples/hana-scale-up-perf-optimized-azure.yaml
Sep 10 16:36:15 vmiscsisrv systemd[1]: /usr/lib/systemd/system/trento-agent.service:7: Executable path is not absolute: trento agent start --consul-config-dir=/srv/consul/consul.d /srv/trento/examples/generic-azure.yaml /srv/trento/examples/hana-scale-up-perf-optimized-azure.yaml
Sep 10 16:36:28 vmiscsisrv systemd[1]: /usr/lib/systemd/system/trento-agent.service:7: Executable path is not absolute: trento agent start --consul-config-dir=/srv/consul/consul.d /srv/trento/examples/generic-azure.yaml /srv/trento/examples/hana-scale-up-perf-optimized-azure.yaml
Sep 10 16:36:33 vmiscsisrv systemd[1]: /usr/lib/systemd/system/trento-agent.service:7: Executable path is not absolute: trento agent start --consul-config-dir=/srv/consul/consul.d /srv/trento/examples/generic-azure.yaml /srv/trento/examples/hana-scale-up-perf-optimized-azure.yaml
Sep 10 16:36:37 vmiscsisrv systemd[1]: /usr/lib/systemd/system/trento-agent.service:7: Executable path is not absolute: trento agent start --consul-config-dir=/srv/consul/consul.d /srv/trento/examples/generic-azure.yaml /srv/trento/examples/hana-scale-up-perf-optimized-azure.yaml

```

Also the extra dash in the sudo command (e.g. `bash -s - ` ) looks like it will be error prone when folks potentially read and type this.  I guess this is redirecting stdin? Maybe add a comment that this is whats happening?

Reply:
The systemd unit is provided by the RPM, @aleksei-burlakov  could you check it please?

Reply:
I also tried the installer syntax `curl -sfL https://raw.githubusercontent.com/trento-project/trento/main/install.sh | sudo bash` on 15 SP2, that also failed:

```
cloudadmin@vmiscsisrv:~> cat /etc/os-release
NAME="SLES"
VERSION="15-SP2"
...

cloudadmin@vmiscsisrv:~> ip a
...
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
    link/ether 00:22:48:84:18:28 brd ff:ff:ff:ff:ff:ff
    inet 10.76.1.4/24 brd 10.76.1.255 scope global eth0
...

cloudadmin@vmiscsisrv:~> curl -sfL https://raw.githubusercontent.com/fabriziosestito/trento/agent-installer/install.sh |sudo bash
Please provide a bind IP for the agent: 10.76.1.4   
Please provide the server IP: 10.0.0.4
Installing trento-agent...
warning: Rebuilding outdated index databases
warning: Generating 18 missing index(es), please wait...
* Installing trento
Done.
You can now start trento-agent with: systemctl start trento-agent
cloudadmin@vmiscsisrv:~> sudo systemctl start trento-agent
Failed to start trento-agent.service: Unit trento-agent.service is not loaded properly: Exec format error.
See system logs and 'systemctl status trento-agent.service' for details.
```

So something different on 15 SP2 vs SP3?  The installer should work on all SLE15, so best test on SP1 and SP0 as well.

Reply:
> Also the extra dash in the sudo command (e.g. bash -s - ) looks like it will be error prone when folks potentially read and type this. I guess this is redirecting stdin? Maybe add a comment that this is whats happening?

This is pretty common tho, see:
https://rancher.com/docs/k3s/latest/en/installation/install-options/how-to-flags/

Reply:
> The systemd unit is provided by the RPM, @aleksei-burlakov could you check it please?

Yes, of course

Reply:
I've changed the rpm, the trento-agent.service will also start in sle 15.2, 15.1 and 15.0